### ISSUE TITLE
redbear arduino DUE shields examples won't compile

### ISSUE THREAD

**Issue by Elmodriver** [NordicSemiconductor/ble-sdk-arduino#24]
I need to use a SAM based arduino for the performance, yet none of the redbear examples compile on the DUE with a EEPROM.H not found.  They do compile as is on the MEGA.  The DUE has no EEPROM.  REDBEAR's  advertising says it works on a DUE.

Redbear support says that it is Nordic software that is failing.  If Redbear can't support the DUE can you suggest another bluetooth solution that will?

**Comment by daviddedwin** [comment]
Yes this is an issue, the #include <EEPROM.h> is included always irrespective of the board. This will prevent the code from running on  the DUE, so you can remove the 

> #include <EEPROM.h>

or wrap it into an AVR specific macro

In addition you will also need to remove the bootloader_setup.cpp and bootloader_setup,h which have EEPROM dependencies.

The examples that do not need the EEPROM will work, i.e. A_Hello_World_Program, UART over BTLE

Examples that need the EEPROM (like the HID examples) will need to have some alternate storage created.

**Comment by Elmodriver** [comment]
I changed the names of the bootloader_setup files and removed the reference
to EEPROM.H.  still doesn't work. numerous  errors relate to memcopy
element tail.

Lets start over.  Is there another manufacturer's product you DO support on
the Due I can use instead?
